File: Application/DropdownLists/DropdownValues.cs
using Application.DTOs;
using Domain;

namespace Application.DropdownLists
{
	public class DropdownValues
	{
		public ICollection<Degree> Degrees { get; set; }
		public ICollection<Region> Regions { get; set; }
		public ICollection<CurrentStatus> CurrentStatuses { get; set; }
		public ICollection<KnowledgeBranchDto> KnowledgeBranches { get; set; }
		public ICollection<Language> Languages { get; set; }
		public ICollection<StudyForm> StudyForms { get; set; }
		public ICollection<Accreditation> Accreditations { get; set; }
	}
}

File: client-app/src/app/models/dropdownValues.ts
import { Degree } from "./degree";
import { KnowledgeBranch } from "./knowledgeBranch";
import { CurrentStatus } from "./profile";
import { Region } from "./region";
import { Language, StudyForm } from "./specialty";
import { Accreditation } from "./university";

export interface DropdownValues {
	degrees: Degree[];
	regions: Region[];
	currentStatuses: CurrentStatus[];
	knowledgeBranches: KnowledgeBranch[];
	languages: Language[];
	studyForms: StudyForm[];
	accreditations: Accreditation[];
}